.circleci/config.yml

```diff
@@ -80,6 +80,16 @@ jobs:
       - run:
           name: rails-6.0_pg-1.2
           command: |
-            ./cc-test-reporter before-build
             bundle exec appraisal rails-6.0_pg-1.2 rspec --format RspecJunitFormatter --format progress
+
+      - run:
+          name: rails-6.1_pg-1.1
+          command: |
+            bundle exec appraisal rails-6.1_pg-1.1 rspec --format RspecJunitFormatter --format progress
+
+      - run:
+          name: rails-6.1_pg-1.2
+          command: |
+            ./cc-test-reporter before-build
```

Comment:
Out of curiosity: why is `./cc-test-reporter before-build` and `after-build` happening in the `rails-6.1_pg-1.2` block, but not others?

Reply:
we only to report test coverage percentage from a single test run. it saves us a smidge of time. i moved it to the most recent version since that seemed to be what we've been as well

Reply:
makes sense! thanks for the details